docs/config/all.py

```diff
@@ -48,4 +48,4 @@
 
 hide_page_contents = ('search', '404', 'contents')
 
-is_prerelease = True
+is_prerelease = false
```

Comment:
```suggestion
is_prerelease = False
```

False need to be capitalized in python